layouts/ucf-resource-link-card.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+/**
+ * Outputs the content of the resource.
+ * Use the `ucf_resource_link_display_card_before` filter
+ * hook to override or modify this output.
+ *
+ * @author RJ Bruneel
+ * @since 1.0.4
+ *
+ * @param $content WP_Post object | The content
+ * @param $args Array | Array of arguments
+ *
+ * @return string | The html to prepend to the output.
+ **/
+if ( ! function_exists( 'ucf_resource_link_display_card_before' ) ) {
+	function ucf_resource_link_display_card_before( $content, $args ) {
```

Comment:
Is the `$content` field used in any of these functions? If not, we can probably just remove it

Reply:
We still need to pass $content in to these layout functions as the first argument, even if it goes unused--each layout hook expects a $content string as the first argument to actually apply modifications to.  Without $content as the first argument, `apply_filters()` won't be able to return a markup string, and themes won't be able to actually apply modifications.